### ISSUE TITLE
Requested featuers

### ISSUE THREAD

**Issue by ilyankou** [OnTheLine/otl-year-built#1]
Jack–

See [this updated version](https://picturedigits.github.io/otl-year-built/) of the map, which I believe addresses all TODOs in your README, and feel free [to merge Picturedigits fork](https://github.com/Picturedigits/otl-year-built) or request further changes.

Ilya

**Comment by JackDougherty** [comment]
Thanks!

**Comment by JackDougherty** [comment]
@ilyankou  - Thanks for rewriting so much of the code! I made some stylistic changes, such as updating colors to blue (selected decade) and gray (all prior decades) and would like your help to finalize some issues:

1) When I tested the map with a new user, they did not understand what colors meant, so I added a legend box, but cannot figure out how to properly place it. Please see https://github.com/OnTheLine/otl-year-built/blob/main/script.js#L154

2) I added westhartford.geojson town boundary, but when it loads it sometimes blocks the pop-up for the parcel boundary layer. I cannot figure out how to fix this. Does it require a "pane" solution? Please see https://github.com/OnTheLine/otl-year-built/blob/main/script.js#L66

3) In the console I see this error for script.js:117
TypeError: null is not an object (evaluating 'l.setStyle')
Initially I thought I caused this by accidentally deleting some JS syntax when revising the variable names, but now I see it's also an error in your version:
https://github.com/Picturedigits/otl-year-built/blob/main/script.js#L103
Please fix

thank you!

**Comment by ilyankou** [comment]
Jack–

1. There were conflicting widths for the legend in CSS classes (`.info` vs `.legend`), so I removed some unnecessary styles.
2. When adding a GeoJSON layer, instructing `interactive: false` will make the layer non-clickable and allow mouse events of the layer underneath, so no need to edit panes.
3. Added an `if` statement to check `l` variable is not null before applying styles.

See pull request with changes.
